libcudacxx/include/cuda/__ptx/ptx_helper_functions.h

```diff
@@ -22,6 +22,7 @@
 #  pragma system_header
 #endif // no system header
 
+#include <cuda/std/__type_traits/enable_if.h>
```

Comment:
Note: we are going to use this in generated functions which will be proposed in subsequent PRs.